docs/source/installation.rst

```diff
@@ -136,52 +136,47 @@ Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support
+                    ``ivadomed`` requires CUDA11 to execute properly. If you have a nvidia GPU, try to look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`_, which needs to be > 3.5 to support CUDA11. Then, make sure to upgrade to nvidia driver to be at least v450+ or newer.
 
-                .. tab:: Mac
+                    If you have a compatible NVIDIA GPU that supports CUDA11 and with the right driver installed try run the following command relevant to your situation:
 
                     .. code::
 
-                       pip install torch==1.8.0 torchvision==0.9.0 --find-links https://download.pytorch.org/whl/torch_stable.html
+                       pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 --find-links https://download.pytorch.org/whl/torch_stable.html
 
-                .. tab:: Repo Installation (Advanced or Developer)
+                .. tab:: CPU Support
 
-                    Run this only if you have already downloaded/cloned the repo with access to the ``requirement.txt`` file, then run the following command while at the repository root level:
+                    If you plan to run ``ivadomed`` on CPU only, install PyTorch per instructions provided below for your specific operating system:
 
-                    .. code::
+                    .. tabs::
 
-                        pip install -r requirements.txt
+                        .. tab:: Windows/Linux
 
-        .. tab:: Nvidia GPU Support
+                            .. code::
 
-            ``ivadomed`` requires CUDA11 to execute properly. If you have a nvidia GPU, try to look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`_, which needs to be > 3.5 to support CUDA11. Then, make sure to upgrade to nvidia driver to be at least v450+ or newer.
+                               pip install torch==1.8.0+cpu torchvision==0.9.0+cpu --find-links https://download.pytorch.org/whl/torch_stable.html
 
-            If you have a compatible NVIDIA GPU that supports CUDA11 and with the right driver installed try run the following command relevant to your situation:
+                        .. tab:: Mac
 
-            .. tabs::
+                            .. code::
 
-                .. tab:: All OS
+                               pip install torch==1.8.0 torchvision==0.9.0 --find-links https://download.pytorch.org/whl/torch_stable.html
 
-                    .. code::
 
-                       pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 --find-links https://download.pytorch.org/whl/torch_stable.html
+        .. tab:: Repo Installation (Advanced or Developer)
 
-                .. tab:: Repo Installation (Advanced or Developer)
+            Run this only if you have already downloaded/cloned the repo with access to the ``requirement_gpu.txt`` file, then run the following command while at the repository root level:
+            .. code::
+
+               pip install -r requirements_gpu.txt
```

Comment:
This `.. code::` bit isn't rendering correctly for me:

![Screenshot from 2022-01-10 13-07-47](https://user-images.githubusercontent.com/16181459/148816626-5ce66f33-9a48-4f48-9d79-537368e8503c.png)

Reply:
Fixed. Thanks!